src/controllers/album.js

```diff
@@ -0,0 +1,101 @@
+const getDb = require("../services/db");
+
+// creates a new album
+exports.create = async (req, res) => {
+  const db = await getDb();
+  const { name, year } = req.body;
+  const { artistId } = req.params;
+  // console.log(artistId);
+
+  try {
+    await db.query(
+      "INSERT INTO Album (name, year, artistId) VALUES (?, ?, ?)",
+      [name, year, artistId]
+    );
+
+    res.sendStatus(201);
+  } catch (err) {
+    console.log(err);
+    res.sendStatus(500).json(err);
+  }
+
```

Comment:
Could delete this empty line here, and all the others above the `db.close()` call in each of the functions below
```suggestion
```